simplq/src/components/pages/Admin/TourSteps.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+export const toursteps = [
+    {
+      selector: '[data-tut="reactour__addMember"]',
+      content:  ()=> (
```

Comment:
Since data-tut is just a selector, can you change the name to something more descriptive like 'reactour-selector'? Just to make it easier for people seeing the code for the first time to understand it better.